core/src/main/java/org/elasticsearch/action/admin/cluster/shards/ClusterSearchShardsResponse.java

```diff
@@ -84,6 +111,20 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             node.toXContent(builder, params);
         }
         builder.endObject();
+        if (indicesAndFilters != null) {
+            builder.startObject("indices");
+            for (Map.Entry<String, AliasFilter> entry : indicesAndFilters.entrySet()) {
+                String index = entry.getKey();
+                builder.startObject(index);
+                AliasFilter aliasFilter = entry.getValue();
```

Comment:
I think it's good to have a summary of all the concrete indices hit by the request anyways, and add the filter when there is one. It is true that we could just skip it if there is none but it feels more consistent if we have the index name in any case?